Powertrain configuration file (PT file) documentation and tutorial notebook (#93)
florentLutz/FAST-OAD-CS23-HE: docs/index.rst

    Models in FAST-OAD-CS23-HE are still a work in progress.

The powertrain builder is a novel feature exclusive to FAST-OAD-CS23-HE, enabling flexible configuration of aircraft

> powertrain architectures. While offering high flexibility, it also introduces additional complexity.

Reviewer: ```suggestion
powertrain architectures. It offers high flexibility in the definition of the propulsive system.
```
